cfgov/paying_for_college/models/blocks.py

```diff
@@ -0,0 +1,57 @@
+from wagtail.wagtailcore import blocks
```

Comment:
I propose moving this file up a level, so it lives at cfgov/paying_for_college/blocks.py. I know we haven't followed this convention, but there's no reason that block types need to live in the models submodule. Moving it out would let you import the blocks (say, for testing) without importing all of Django's model code, which in turn loads settings, etc.

Reply:
√